shared/@hedvig-ui/JsonSchemaForm/json-schema-form.tsx

```diff
@@ -86,11 +85,15 @@ const ArrayFieldTemplate: React.FC<ArrayFieldTemplateProps> = ({
           <ItemWrapper key={element.key}>
             <ItemTitleWrapper>
               <strong>{index + 1}.</strong>
-              <ItemRemoveButton onClick={element.onDropIndexClick(index)}>
+              <Button
+                variation={'icon'}
```

Comment:
As a general comment that I don't want to do everywhere, but I think this should be the preferred approach:
```suggestion
                variation="icon"
```

Reply:
agreed, saw that in a few places as well :)